manifests/charts/gateways/istio-ingress/templates/service.yaml

```diff
@@ -1,5 +1,5 @@
 {{ $gateway := index .Values "gateways" "istio-ingressgateway" }}
-{{- if not $gateway.customService }}
+{{- if or (eq $gateway.customService true) (eq $gateway.revisionService true) }}
```

Comment:
why do we need `customService` and `revisionService`?

Reply:
we dont. I didn't want to remove customService as it was in a released chart, but I did want consistency with revisionDeployment.

Reply:
I think there may be a better way to determine if a new deployment should be triggered without adding a new configuration flag. I'll have to think it through a little more, but I'm not certain - other than from a mergability perspective (i.e. your concern in this review) - it is relevant for the RFC.